src/Build/Graph/ProjectGraph.cs

```diff
@@ -12,12 +13,19 @@ namespace Microsoft.Build.Graph
     /// </summary>
     internal sealed class ProjectGraph
     {
+        private readonly List<ProjectGraphNode> _projectNodes = new List<ProjectGraphNode>();
+
         /// <summary>
         /// Constructs a graph starting from the given project file.
         /// </summary>
         /// <param name="entryProjectFile">The project file to use as the entry point in constructing the graph</param>
         /// <exception cref="InvalidProjectFileException">If the evaluation of any project in the graph fails.</exception>
-        public ProjectGraph(string entryProjectFile) { }
+        public ProjectGraph(string entryProjectFile)
+        {
+            var graphNode =
+                new ProjectGraphNode(ProjectCollection.GlobalProjectCollection.LoadProject(entryProjectFile));
```

Comment:
Eventually we'll need to probably take a project collection (optionally), but for now I think this is just fine to get things moving. We can worry about all the "right" overloads to have after the basic functionality takes shape.

Reply:
Another potential overload is one accepting an EvaluationContext. If none is given, we should probably instantiate and use our own.

Speaking of that, should we load the project via the project collection or via the `Project.FromFile` static factory? The latter provides more configuration via [ProjectOptions](https://github.com/Microsoft/msbuild/blob/master/src/Build/Definition/ProjectOptions.cs).

But for now, this way is probably good.